bin/import_cv2.py

```diff
@@ -52,51 +84,19 @@ def _maybe_convert_set(input_tsv, audio_dir, label_filter, space_after_every_cha
     with open(input_tsv, encoding='utf-8') as input_tsv_file:
         reader = csv.DictReader(input_tsv_file, delimiter='\t')
         for row in reader:
-            samples.append((row['path'], row['sentence']))
+            samples.append((path.join(audio_dir, row['path']), row['sentence']))
```

Comment:
I think this was omitted on purpose. CSVs should have relative paths so that the snakepit mount points can be moved around without breaking the CSVs.

Reply:
Are you sure about that? All the importers here would produce absolute path CSVs.

Reply:
@tilmankamp knows for sure. As far as I know they're supposed to all be relative.

Reply:
@reuben I've just moved the `os.path.join` from inside the `one_sample` to before, but as much as I can tell I have not changed the logic itself.

Reply:
@lissyx then I guess this is not a concern.